### PR TITLE
Ds 182/footer

### PR DESCRIPTION
**Pull request by AnthonyA44**
# DS-182-footer

Main Reviewer: @Caterina-vanHerpt 

#### Description & Background Context:

Current footer developed based on figma designs provided by the design team

#### Where should the reviewer start?

footer.css & Footer.html files under src/components/Footer

File name: change

#### How should this be tested?

If appropriate...

#### Are there any breaking changes?

N/A

### CODE REVIEW

**Review by Caterina-vanHerpt — COMMENTED** [review]
Looks great! 

Let me know if you have any questions.

File: src/components/Footer/static/Footer.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+
+<!-- When using this html, make sure to includes these links -->
+
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+  <meta content="utf-8" http-equiv="encoding">
+  <!-- google fonts load these in at project level -->
```
[the remark this answers is not in the document]
Comment (AnthonyA44) [reply src/components/Footer/static/Footer.html]: Added Developer Notes to comment

File: src/components/Footer/static/Footer.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Noto+Sans:wght@400;700&family=Rubik:wght@500&display=swap"
+    rel="stylesheet">
+  <link href="footer.css" rel="stylesheet" type="text/css" />
+</head>
+
+<!-- html for the footer -->
+<footer class="footer-container">
+  <div class="footer-top">
+    <!-- Replace with your date modified component -->
+    <span>Date modified: 2021-02-27</span>
```
[the remark this answers is not in the document]
Comment (AnthonyA44) [reply src/components/Footer/static/Footer.html]: I fixed the semantics to follow Canada.ca. Is this how you intended for us to us Developer Notes?

File: src/components/Footer/static/Footer.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Noto+Sans:wght@400;700&family=Rubik:wght@500&display=swap"
+    rel="stylesheet">
+  <link href="footer.css" rel="stylesheet" type="text/css" />
+</head>
+
+<!-- html for the footer -->
+<footer class="footer-container">
+  <div class="footer-top">
```
[the remark this answers is not in the document]
Comment (AnthonyA44) [reply src/components/Footer/static/Footer.html]: added h2 and changed the tag to section

File: src/components/Footer/static/Footer.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    <!-- Replace with your date modified component -->
+    <span>Date modified: 2021-02-27</span>
+  </div>
+  <div class="breakpoint"></div>
```
[the remark this answers is not in the document]
Comment (AnthonyA44) [reply src/components/Footer/static/Footer.html:32]: I've changed all the divs you've pointed out to have section tags instead.

File: src/components/Footer/static/footer.css [hunk 5]
```diff
@@ -0,0 +1,186 @@
+a {
```
[the remark this answers is not in the document]
Comment (AnthonyA44) [reply src/components/Footer/static/footer.css]: Moved styles into their own classes

File: src/components/Footer/static/footer.css [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  height: fit-content;
+  padding: 0 20px 0 20px;
+  margin: 0;
+}
+
+/* button styling */
+.btn {
```
[the remark this answers is not in the document]
Comment (AnthonyA44) [reply src/components/Footer/static/footer.css:49]: Removed the div and added the styling directly to the a tag

File: src/components/Footer/static/Footer.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (Caterina-vanHerpt) [reply src/components/Footer/static/Footer.html]: Yes, looks good to me!

File: src/components/Footer/static/Footer.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (Caterina-vanHerpt) [reply src/components/Footer/static/Footer.html:32]: We shouldn't have a section tag nested in another section tag. The way you can think about section tags is that wherever we have put a heading, there should be a section tag. If we need to wrap this section in a div, or have divs insides the section for styling purposes then that is ok. 

My comment here was in regards to the empty breakpoint divs, we could add the class "breakpoint" to the section or div elements that have content instead and remove the `<div class="breakpoint"></div>`

File: src/components/Footer/static/footer.css [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+  padding-top: 16px;
+}
+
+/* link styling */
+.link-section {
+  display: flex;
+  flex-direction: column;
+}
+.link-text {
+  font-family: Noto Sans;
```
[the remark this answers is not in the document]
Comment (Caterina-vanHerpt) [reply src/components/Footer/static/footer.css]: Can you double check that all of our font-family styles have a backup?

[the remark this answers is not in the document]
Comment (AnthonyA44) [reply src/components/Footer/static/footer.css]: Backup fonts have been added to all font-family

File: src/components/Footer/static/Footer.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (AnthonyA44) [reply src/components/Footer/static/Footer.html]: breakpoint divs have been removed and styling of breakpoint class has been moved to the appropriate section tags

[the remark this answers is not in the document]
Comment (AnthonyA44) [reply src/components/Footer/static/Footer.html]: Also removed nested section tags*

**Review by Caterina-vanHerpt — APPROVED** [review]

**Pull request merged**